ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/rules/TrailingCommaOnDeclarationSiteRule.kt

```diff
@@ -306,7 +303,12 @@ public class TrailingCommaOnDeclarationSiteRule :
                     val leafBeforeArrowOrNull = leafBeforeArrowOrNull()
                     val addNewLine =
                         leafBeforeArrowOrNull
-                            ?.let { !(leafBeforeArrowOrNull is PsiWhiteSpace && leafBeforeArrowOrNull.textContains('\n')) }
+                            ?.let {
+                                !(
+                                    leafBeforeArrowOrNull.elementType == ElementType.WHITE_SPACE &&
+                                        leafBeforeArrowOrNull.textContains('\n')
+                                )
+                            }
                             ?: false
```

Comment:
This can be simplified to:
```
                    val addNewLine = !(leafBeforeArrowOrNull?.isWhiteSpaceWithNewline() ?: true)
```

but it requires another change (see next review comment) to resolve compilation errors.